internal/utils/utils_test.go

```diff
@@ -55,44 +52,6 @@ func TestCheckVersion(t *testing.T) {
 	}
 }
 
-func TestParseCiliumVersion(t *testing.T) {
```

Comment:
Okay, I don't really want to test someone else's code, but I still think it might be useful for us to test our expected values with `semver.ParseTolerant`, just to ensure the values we pass to it actually work as expected.

Reply:
sure i'll add some tests ✅